src/network/serverpackethandler.cpp

```diff
@@ -1277,21 +1279,19 @@ void Server::handleCommand_Interact(NetworkPacket* pkt)
 				}
 			}
 
-			/* Actually dig node */
+			if (!is_valid_dig || n.getContent() == CONTENT_IGNORE)
```

Comment:
Resend the block is it's not a valid dug node -> cheat prevention.

Reply:
I've changed it.